utils/doclint/generate_types/index.js

```diff
@@ -19,7 +19,6 @@ const objectDefinitions = [];
     path.join(__dirname, '..', '..').length
   )}
 import { ChildProcess } from 'child_process';
-import { EventEmitter } from 'events';
```

Comment:
this is a quick fix to get this branch passing CI. On doing so I realised this file is not needed so https://github.com/puppeteer/puppeteer/pull/5932 removes it completely.